src/data_loading.jl

```diff
@@ -8,6 +8,18 @@ struct ECProfiles <: Dataset end
 struct MetabolicProfiles <: Dataset end
 struct Neuroimaging <: Dataset end
 
+function map_string_timepoint(concatstring)::Tuple{Int64, Int64}
+    letter_to_timepoint = Dict( [ x => y for (x,y) in zip('a':'l',1:14)] ) 
+    isletter(concatstring[end]) ? (return(parse(Int64, concatstring[1:end-1]), letter_to_timepoint[concatstring[end]])) : (return(parse(Int64, concatstring), 1))
+end # end function
+
+function split_subject_timepoint(stringsvector)
+    mappings = map(map_string_timepoint, stringsvector)
+    subjects = map(x -> x[1], mappings)
+    timepoints = map(x -> x[2], mappings)
+    return subjects, timepoints
+end # end function
```

Comment:
See `split_tp()` function in `wrangle.jl`: https://github.com/Klepac-Ceraj-Lab/Resonance/blob/main/src/wrangle.jl#L1-L11

Can just do `split_tp.(stringsvector)`